src/components/popup/NewProductAdmin.js

```diff
@@ -13,14 +13,17 @@ export default class NewProductAdmin extends React.Component {
     this.state = {
       categories: [],
       subCategories: [],
-      suppliers: []
+      suppliers: [],
+      category: 1,
+      quantity: 0,
+      description: ""
```

Comment:
přesuň celý state mimo constructor a constructor smaž (při přesouvání odmaž this aby to správně fungovalo)